### ISSUE TITLE
Have problem on Firefox

### ISSUE THREAD

**Issue by sangdth** [CharlesStover/use-clippy#10]
```
08:57:56.210 TypeError: nav.clipboard.readText is not a function use-clippy.js:130
```
I'm using React 16.8 and Node 10.16, Firefox 69

**Comment by CharlesStover** [comment]
Thanks for the report! That is an interesting error. For documentation's sake, Mozilla's definition of the Clipboard API explicitly defines [navigator.clipboard.readText](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API).

Can you share the following by manually executing it in your Firefox console?

```
console.log(navigator.clipboard);
```

**Comment by sangdth** [comment]
@CharlesStover I don't trust their documentation.
Here is the screenshot, I can not copy the object, it always gives me an empty object.
<img width="559" alt="Screenshot 2019-10-09 at 20 54 16" src="https://user-images.githubusercontent.com/1083478/66507402-cf66cd80-ead7-11e9-996a-1f03bf46918d.png">


Btw, the `readText` is not there. 
````
console.log(typeof navigator.clipboard.readText)
>> undefined
````

For example, in [here](https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query#Browser_compatibility) they stated that the permission query options available, but in reality, if you call `navigator.permissions.query({ name: "clipboard-read" })` it will throw an exception.

**Comment by CharlesStover** [comment]
This should be resolved in 1.0.4, where it will now fail silently if readText is not supported. Even TypeScript's clipboard interface does not correctly identify that readText is optional. 😵
